pkg/scalers/scaler.go

```diff
@@ -79,6 +79,8 @@ func GetFromAuthOrMeta(config *scalersconfig.ScalerConfig, field string) (string
 		result = config.AuthParams[field]
 	} else if config.TriggerMetadata[field] != "" {
 		result = config.TriggerMetadata[field]
+	} else if value, ok := config.AuthParams["scheme"]; ok && strings.Contains(value, "mongodb+srv") {
+		return result, err
```

Comment:
This method is generic for all the scalers, we should handle mongo specific configs here. Pleae move this parsing logic to the scaler 🙏

Reply:
have moved this to `mongo_scaler.go` file